WallProject/WallProject/Services/Serives Implementations/PersonService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using WallProject.Models;
+using WallProject.Services.Services_Interfaces;
+
+namespace WallProject.Services.Serives_Implementations
+{
+    public class PersonService : IPersonService
+    {
+        //Dobra praktyka - pomaga uniknac problemu z wyczerpaniem gniazda
+        private readonly IHttpClientFactory _clientFactory;
+        public PersonService(IHttpClientFactory clientFactory)
+        {
+            _clientFactory = clientFactory;
+        }
+
+        async public Task<ServiceResult<PersonViewModel>> getById(int userID)
+        {
+            var client = _clientFactory.CreateClient("webapi");
+            var result = await client.GetAsync($"person/{userID}");
+            var jsonString = await result.Content.ReadAsStringAsync();
+
+            return new ServiceResult<PersonViewModel>(jsonString, result.StatusCode);
```

Comment:
bardzo ładny kod w tym serwisie, szkoda że w innych się nie udało niczego wymyślić

Reply:
żeby to "uładnić" konieczne byłoby w w jakiś dziki sposób wywoływanie serwisów gdzieś w maperach albo w ServiceResult.
Nie udało mi się więc tego w żaden sposób uładnić